Forbidden pass context with "beginMemory" or "beginTime" keys to `Profiler::end()` + Validate category + Psalm 1 (#42)
yiisoft/profiler: CHANGELOG.md

# Profiler Change Log

## 1.0.2 under development

> - Chg #42: Forbidden pass context with "beginMemory" or "beginTime" keys to `Profiler::end()` (vjik)

Reviewer: ```suggestion
- Bug #42: Forbid passing context with "beginMemory" or "beginTime" keys to `Profiler::end()` (vjik)
```

Reviewer 2: fixed

> - Bug #42: Check that category of message is a string (vjik)

Reviewer: ```suggestion
- Bug #42: Check that category of a message is a string (vjik)
```

> - Bug #42: Checking that category of message is a string (vjik)

Reviewer: ```suggestion
- Bug #42: Check that category of message is a string (vjik)
```
